src/liboslexec/llvm_gen.cpp

```diff
@@ -386,6 +391,17 @@ LLVMGEN (llvm_gen_printf)
         }
     }
 
+    if (rop.use_optix() && arg > (format_arg + 2)) {
+        ASSERT (0 && "OptiX printf only supports 0 or 1 arguments at this time");
```

Comment:
Can we make this into some kind of warning and turn the printf into something harmless, rather than an assertion that will bring down the whole app?

Reply:
Definitely. If you find other instances, let me know and I will update them.

Reply:
Also, I just had the idea that I can do some constant folding of printf in the optimizer, which I bet in a wide array of cases can turn printfs into either constant strings or just one argument left.